README.md

```diff
@@ -62,6 +62,13 @@ You may need to wait up to 10 minutes for the changes to propagate.
 
 These steps based on [directions at the omniauth-google-oauth2 gem](https://github.com/zquestz/omniauth-google-oauth2#google-api-setup).
 
+### Local Eyeshade Setup
+
+1. Follow the [setup instructions](https://github.com/brave-intl/bat-ledger) for bat-ledger
+2. Add `export API_EYESHADE_BASE_URI="http://127.0.0.1:3002"` to your secrets script
+3. Add `export API_EYESHADE_KEY="00000000-0000-4000-0000-000000000000"` to your secrets script
+4. In [config/secrets.yml](https://github.com/brave/publishers/blob/master/config/secrets.yml), under `development:` set `api_eyeshade_offline` to `false`
```

Comment:
i think we can remove the default `api_eyeshade_offline`, and instead if api_eyeshade_base_uri is not set then assume we want an offline eyeshade.